CMakeLists.txt

```diff
@@ -765,7 +765,7 @@ if (USE_BREAKPAD)
     endif()
 endif()
 
-option(PREFER_EXTERNAL_LIBS "Tries to use system libs where possible." ON)
+option(PREFER_EXTERNAL_LIBS "Tries to use system libs where possible." OFF)
```

Comment:
This shouldn't be changed because Unvanquished doesn't even build in the default configuration that way (it defines the Freetype lib twice).

Reply:
The submodule Freetype build bug is now fixed.

When building both the engine and a native game, the native game now reuses the Freetype target already sets by the engine.

Reply:
> The Freetype submodule double build bug is now fixed.

Great!

What's the motivation for the change though? It makes builds longer and people building Daemon themselves can no longer benefit from distro security updates.